lib/keystores.js

```diff
@@ -16,6 +16,10 @@ require('./config');
 const api = {};
 module.exports = api;
 
+api._helpers = {
```

Comment:
I don't think we should introduce this new pattern. I also recommend we remove `api = {}` here and do `exports.functionName = function` below as we've discovered the `api = {}` pattern is problematic. Private functions start with `_` and that should be sufficient. If we wanted to namespace things up we'd add a `helpers.js` file, but I don't think we need to at this time.